packages/examples/cvat/exchange-oracle/src/handlers/job_creation.py

```diff
@@ -135,6 +137,42 @@ def add_message(self, message: str, *, sample_id: str, sample_subset: str):
         )
 
 
+def setup_gt_job_for_cvat_task(
+    task_id: int,
+    gt_dataset: dm.Dataset,
+    *,
+    dm_export_format: str = "coco",
+) -> None:
+    dm_format_to_cvat_format = {
+        "coco": "COCO 1.0",
+        "cvat": "CVAT 1.1",
+    }
+
+    dm_format_to_annotations_path = {
+        "coco": "annotations/instances_default.json",
+        "cvat": "default.xml",
+    }
+
+    task_status: cvat_api.UploadStatus | None = None
+
+    while task_status != cvat_api.UploadStatus.FINISHED:
+        task_status, _ = cvat_api.get_task_upload_status(task_id)
+        if not task_status or task_status == cvat_api.UploadStatus.FAILED:
+            return  # will be handled in state_trackers.py::track_task_creation
+
+    if task_status == cvat_api.UploadStatus.FINISHED:
+        with TemporaryDirectory() as tmp_dir:
+            gt_dataset.export(
+                save_dir=tmp_dir, save_images=False, format=dm_export_format
+            )  # reindex=True ?
```

Comment:
Reindex will change image and annotation ids. If you're not going to use this dataset for anything except uploading, and it's possible that annotation ids are repeated, you can use the option.

Reply:
Updated in https://github.com/humanprotocol/human-protocol/pull/2730